amq-streams/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Edit the CSV to disable FIPS_MODE
+  kubernetes.core.k8s:
+    state: patched
+    namespace: "{{ install_namespace }}"
+    name: "{{ channel.name }}"
```

Comment:
this will return the channel name, not the CSV name. 

`channel.currentCSV` should work.

Reply:
Ooops... Meant to use that. Somehow it still worked, but probably not ideal. I'll fix.